lib/ansible/playbook/task.py

```diff
@@ -70,11 +70,11 @@ class Task(Base, Conditional, Taggable, Become):
 
     _any_errors_fatal     = FieldAttribute(isa='bool')
     _async                = FieldAttribute(isa='int', default=0)
-    _changed_when         = FieldAttribute(isa='string')
+    _changed_when         = FieldAttribute(isa='list', default=[])
     _delay                = FieldAttribute(isa='int', default=5)
```

Comment:
doesn't this style of initializer on FieldAttribute cause the instance aliasing issue (eg, where all instances end up sharing the same empty list)?